src/__tests__/migrate.ts

```diff
@@ -190,6 +190,61 @@ test("with pool client", async (t) => {
   }
 })
 
+test("with custom migration table name", async (t) => {
+  const databaseName = "migration-test-custom-migration-table"
+  const dbConfig = {
+    database: databaseName,
+    user: "postgres",
+    password: PASSWORD,
+    host: "localhost",
+    port,
+  }
+
+  const migrateWithCustomMigrationTable = () =>
+    migrate(dbConfig, "src/__tests__/fixtures/success-first", {
+      migrationTableName: "my_migrations",
+    })
+
+  await createDb(databaseName, dbConfig)
+  await migrateWithCustomMigrationTable()
+
+  t.truthy(await doesTableExist(dbConfig, "my_migrations"))
+  t.truthy(await doesTableExist(dbConfig, "success"))
+
+  await migrateWithCustomMigrationTable()
+})
+
+test("with custom migration table name in a custom schema", async (t) => {
```

Comment:
Does this test satisfy what you were asking for in https://github.com/ThomWright/postgres-migrations/pull/33#issuecomment-571604321?